src/pages/app/pages/PostInstall.tsx

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { Button } from '@/pages/shared/components/ui/Button';
+import {
+  CaretDownIcon,
+  CheckIcon,
+  ExternalIcon,
+} from '@/pages/shared/components/Icons';
+import { getBrowserName, type BrowserName } from '@/shared/helpers';
+import { useBrowser, useTranslation } from '@/app/lib/context';
+
+export const Component = () => {
+  const browser = useBrowser();
+
+  return (
+    <div
+      className="flex h-screen flex-col items-center overflow-hidden landscape:justify-center"
+      style={{
+        backgroundImage: `url("/assets/images/bg-tile.svg")`,
+        backgroundSize: '40vmax',
+      }}
+    >
+      <div className="flex max-h-full w-full max-w-screen-2xl grid-flow-col-dense flex-col content-between items-center gap-6 p-8 landscape:grid landscape:p-4">
+        <Header />
+        <Main openPopup={() => browser.action.openPopup({})} />
+      </div>
+    </div>
+  );
+};
+
+const Header = () => {
+  const t = useTranslation();
+  return (
+    <div className="text-center">
+      <img
+        src="/assets/images/logo.svg"
+        className="mx-auto mb-4 w-16 text-center landscape:w-36 landscape:2xl:w-48"
+        alt=""
+      />
+      <p className="text-xl font-bold text-secondary-dark landscape:mb-2 landscape:text-3xl landscape:2xl:mb-3 landscape:2xl:text-4xl">
+        {t('tagline_text_1')}
+      </p>
+      <p className="text-xl font-light text-secondary-dark landscape:text-3xl landscape:2xl:text-4xl">
+        {t('tagline_text_2')}
+      </p>
+    </div>
+  );
+};
+
+const Main = ({ openPopup }: { openPopup: () => Promise<void> }) => {
+  const t = useTranslation();
+  return (
+    <div className="mx-auto flex h-full w-full max-w-2xl flex-col gap-6 overflow-hidden rounded-lg border border-gray-200 bg-gray-50/75 p-8 shadow-md backdrop-blur-0">
+      <h2 className="rounded-2xl bg-gray-100 p-4 text-center text-lg font-medium">
+        {t('postInstall_text_title')}
+      </h2>
+
+      <div className="h-full overflow-y-auto">
+        <Steps />
+      </div>
+
+      <div className="ml-auto mt-auto">
+        <Button onClick={openPopup}>{t('postInstall_action_submit')}</Button>
+      </div>
+    </div>
+  );
+};
+
+const Steps = () => {
+  const browser = useBrowser();
+  const t = useTranslation();
+  const isPinnedToToolbar = usePinnedStatus();
+  const [isOpen, setIsOpen] = React.useState(0);
+  const browserName = getBrowserName(browser, navigator.userAgent);
+
+  const onClick = React.useCallback((index: number, open: boolean) => {
+    setIsOpen((prev) => (!open ? index : prev + 1));
+  }, []);
+
+  return (
+    <ol className="flex flex-col gap-4">
+      <Step
+        index={0}
+        open={isOpen === 0}
+        onClick={onClick}
+        title={
+          <>
```

Comment:
really smol nitpick: React.Fragment instead of `<>` or vice-versa for consistency